content/integrations/faq/why-is-the-kubernetes-check-failing-with-a-connecttimeout-error-to-port-10250.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Why is the Kubernetes check failing with a ConnectTimeout error to port 10250?
+kind: faq
+customnav: integrationsnav
+---
+
+The agent assumes that the kubelet API is available at the default gateway of the container. If that's not the case because you are using a software defined networks like Calico or Flannel, the agent needs to be specified using an environment variable:
```

Comment:
s/software defined networks/software defined network/

The Agent needs to be specified? Or does this really mean that the kubelet API host needs to be specified?